igor_src/main.cpp

```diff
@@ -45,9 +45,24 @@
 #include <chrono>
 #include <set>
 
+#include <string>
+#include "CDR3SeqData.h"
+#include "ExtractFeatures.h"
 
 using namespace std;
 
+
+// FIXME: PUT THIS IN ANOTHER FILE
+
+
+// TODO: Possible typedef definitions for code readability.
+typedef std::string strSeqID;    // fasta description >strSeqID
+typedef std::string strSequence;
+typedef vector< pair<const int, const strSequence> > VectorIndexedSeq;
+typedef pair<strSeqID, strSequence> PairSeq; // FIXME: Not a good name convention.
```

Comment:
```suggestion
typedef pair<strSeqID, strSequence> Indexed_Seq; // A typedef for a doublet of a strSequence and the corresponding sequence index
```
does it sound good to you?